frontend/src/Notes.js

```diff
@@ -0,0 +1,65 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+
+export default function NoteDescription({
+  notes,
+  index,
+  completeNote,
+  removeNote,
+}) {
+  return (
+    <main>
+      <MyNotes
+        style={{
+          textDecorationThickness: notes.isCompleted ? '0.1em' : '',
+        }}
+        notes={notes}
+      >
+        <label className="checkbox" htmlFor="checkbox">
+          <Checkbox onClick={() => completeNote(index)} type="checkbox" />
+        </label>
+        <span className="note">{notes.value}</span>
+        <DescriptionButton onClick={() => removeNote(notes)}>
+          <i className="fas fa-trash-alt"></i>
+        </DescriptionButton>
+      </MyNotes>
+    </main>
+  )
+}
+
+const Checkbox = styled.input`
+  width: 13px;
+  height: 13px;
+  background-color: transparent;
+  color: #f25e5e;
+  transition: all 0.2s ease-in-out;
+  border: 2px solid #ececec;
+  border-radius: 0.25rem;
+`
+
+const MyNotes = styled.section`
+  text-decoration: ${(props) =>
+    props.notes.isCompleted ? 'line-through' : ''};
```

Comment:
du hast einmal oben ein style mit conditional styling und hier unten auch nochmal. Ich würde das von oben hier unten in das styling mit reinnehmen, dann ist es etwas übersichtlicher.

Reply:
Die styled-components kennen die property: textDecorationThickness nicht. Ich muss es leider so lassen.